src/odemis/acq/stream/_base.py

```diff
@@ -1056,5 +1056,5 @@ def getRawMetadata(self):
         Gets the raw metadata structure from the stream.
         A list of metadata dicts is returned.
         """
-        return [data.metadata for data in self.raw if isinstance(data, model.DataArray)]
+        return [data.metadata for data in self.raw if isinstance(data, (model.DataArray, model.DataArrayShadow))]
```

Comment:
That's not what I meant. In case there is a DAS, `.raw` doesn't contain really the raw data, it contains the raw data of the tiles in cache. I think you need to change it this way. Eventually, in `gui.cont.streams` you should be able to update `_display_metadata()` to just have `self.stream.getRawMetadata()[0]`.
```
if hasattr(self, '_das'):
      return [self._das.metadata]
else:
     return [None if data is None else data.metadata for data in self.raw]
```